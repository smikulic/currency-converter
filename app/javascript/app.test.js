import { mount } from '@vue/test-utils'
import App from './app.vue'

describe('App.vue', () => {
  it('increments counter', () => {
    const wrapper = mount(App);

    expect(wrapper.html()).toMatchSnapshot()
  })
})
