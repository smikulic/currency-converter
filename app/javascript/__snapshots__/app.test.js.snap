// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue increments counter 1`] = `
<div id="app-container">
  <div class="row">
    <div class="date-picker"><input type="date"></div>
  </div>
  <div class="row">
    <div class="col"><select class="select-currency">
        <option value="USD">
          USD
        </option>
        <option value="EUR">
          EUR
        </option>
        <option value="NOK">
          NOK
        </option>
        <option value="HRK">
          HRK
        </option>
        <option value="GBP">
          GBP
        </option>
        <option value="RUB">
          RUB
        </option>
        <option value="AUD">
          AUD
        </option>
        <option value="CZK">
          CZK
        </option>
        <option value="INR">
          INR
        </option>
        <option value="JPY">
          JPY
        </option>
        <option value="CNY">
          CNY
        </option>
      </select> <input type="text" class="input-currency-from"></div>
    <div class="col"><span class="icon-swap"><!----></span></div>
    <div class="col"><select class="select-currency">
        <option value="USD">
          USD
        </option>
        <option value="EUR">
          EUR
        </option>
        <option value="NOK">
          NOK
        </option>
        <option value="HRK">
          HRK
        </option>
        <option value="GBP">
          GBP
        </option>
        <option value="RUB">
          RUB
        </option>
        <option value="AUD">
          AUD
        </option>
        <option value="CZK">
          CZK
        </option>
        <option value="INR">
          INR
        </option>
        <option value="JPY">
          JPY
        </option>
        <option value="CNY">
          CNY
        </option>
      </select> <input type="text" disabled="disabled" class="input-currency-to"></div>
  </div>
  <div class="row"><span class="message">Select currencies to convert!</span></div>
</div>
`;
